src/sass/components/_network.scss

```diff
@@ -1,12 +1,44 @@
-.network-btn {
-  display: inline-block;
-  max-width: 240px;
-  margin-right: $space-sm;
-  margin-left: $space-sm;
+$full-width: 100%;
```

Comment:
Just FYI, part of the point of requiring variables for style declarations is to try and avoid so many custom values. There are a number of things here that seem unnecessarily customized or are overly specific (in a way that looks very brittle). See what you can use of the existing spacing variables with a goal of removing as many of these custom variables as possible. Feel free to ping me if you have questions.